### PR TITLE
Additional small fixes across the repo

### PR DESCRIPTION
**Pull request by Staudey**
## Description

- Global
  - Small style/typo fixes, use sentence case for headers (except page titles), update links
- Boot Management
  - Fix headers
- Budgie
  - Leave alone in favour of in-progress PR
- GNOME Configuration
  - Fix headers
  - Remove no longer existing video
- MATE Configuration
  - Fix headers
- Plasma Configuration
  - Fix headers
- Getting Involved
  - Use link to translations help page instead of non-functioning translate.getsol.us
- Java
  - Add version 21
- IBus
  - Use correct IME package names

### CODE REVIEW

File: docs/user/contributing/getting-involved.md [hunk 1]
```diff
@@ -67,4 +67,4 @@ We're always looking to improve our systems, especially when they're not functio
 
 ## Translations
 
-Assistance is always welcome in translating our projects [to other languages](https://translate.getsol.us)!
+Assistance is always welcome in translating our projects [to other languages](/docs/packaging/translation-instructions)!
```
Comment (davidjharder) [inline docs/user/contributing/getting-involved.md:70]: Do you know where `translate.getsol.us` used to point? We could point it to `https://app.transifex.com/solus/`

**Review by davidjharder — APPROVED** [review]
Thanks for a very thorough cleanup

File: docs/user/contributing/getting-involved.md (hunk 1 above)
Comment (Staudey) [reply docs/user/contributing/getting-involved.md:70]: https://web.archive.org/web/20220520013432/https://translate.getsol.us/

The Weblate dashboard apparently. So yeah, the transifex link would be the up-to-date equivalent.

**Comment by Staudey** [comment]
The issue with the https version of oscollective.org was resolved, so changing it to that (removing the superfluous `www.` though)

File: docs/user/contributing/getting-involved.md (hunk 1 above)
Comment (davidjharder) [reply docs/user/contributing/getting-involved.md:70]: DNS change is not straightforward, we'll leave it as is in your revision.

**Pull request merged**